src/App.js

```diff
@@ -1,15 +1,77 @@
-import SearchBar from './components/SearchBar/SearchBar';
-import Users from './components/Users/Users';
-import './App.css';
+import { useEffect, useState } from "react";
+import { Button } from "react-bootstrap";
+import SearchBar from "./components/SearchBar/SearchBar";
+import Users from "./components/Users/Users";
+import axios from "axios";
+import "./App.scss";
 
 function App() {
+  const API = process.env.REACT_APP_API_URL;
+  const [users, setUsers] = useState([]);
+  const [searchInput, setSearchInput] = useState("");
+  const [expanded, setExpanded] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState("");
+
   // TODO: Fetch data here
+  useEffect(() => {
+    getUsers();
+  }, []); // eslint-disable-line
+
+  const getUsers = async () => {
+    try {
+      setError("");
+      setLoading(true);
+      await axios
+        .get(`${API}/users`)
+        .then((res) => {
+          setLoading(false);
+          setUsers(res.data.data);
+        })
+        .catch((err) => {
+          setLoading(false);
+          setError(err);
+        });
+    } catch (err) {
```

Comment:
I don't know axios as well, but do you need the `try/catch` block here if you already have `.catch`?

Reply:
I don't think i do. I added it just in case.